django/db/models/base.py

```diff
@@ -1630,9 +1630,39 @@ def _check_ordering(cls):
         # Convert "-field" to "field".
         fields = ((f[1:] if f.startswith('-') else f) for f in fields)
 
-        # Skip ordering in the format field1__field2 (FIXME: checking
-        # this format would be nice, but it's a little fiddly).
-        fields = (f for f in fields if LOOKUP_SEP not in f)
+        # seprate related field and non related fields
+        _fields = []
+        related_fields = []
+        for f in fields:
+            if LOOKUP_SEP in f:
+                related_fields.append(f)
+            else:
+                _fields.append(f)
+        fields = iter(_fields)
+        related_fields = iter(related_fields)
+
+        # check related fields
+        for field in related_fields:
+            parts = field.split(LOOKUP_SEP)
+            _cls = cls
+            fld = None
+            for part in parts:
+                try:
+                    fld = _cls._meta.get_field(part)
+                    if fld.is_relation:
+                        _cls = fld.get_path_info()[-1].to_opts.model
+                    else:
+                        _cls = None
+                except (FieldDoesNotExist, AttributeError):
+                    if fld is None or fld.get_transform(part) is None:
+                        errors.append(
+                            checks.Error(
+                                "Invalid 'ordering' item '%s'." % field,
```

Comment:
I think this should probably be `"'ordering' refers to the nonexistent related field or lookup '%s'." % field` to match the other error messages.